ru/turing/courses/lesson2/Solution.java

```diff
@@ -1,4 +1,38 @@
 package ru.turing.courses.lesson2;
 
+import java.util.Scanner;
+
 public class Solution {
+    public static void main(String[] args) {
+        baza Person = new baza();
```

Comment:
```suggestion
        Person person= new Person();
```
У классов должны быть разумные названия, из названия baza не очень понятно, о чем этот класс. А еще названия классов пишутся с заглавной, а методов и полей -- с маленькой